tests/test_shadow_fork.py

```
@@ -72,38 +72,53 @@ def test_create_fails_if_already_active(self, tmp_path):
+                mock_git.side_effect = [
+                    (True, "main", ""),  # Get current branch
+                    (True, "", ""),  # Check for uncommitted changes
+                    (True, "", ""),  # Create and switch to shadow branch
+                ]
+
+                # Create hooks directory for the test
+                (tmp_path / ".git" / "hooks").mkdir(parents=True)
 
-            with pytest.raises(RuntimeError, match="does not exist"):
-                fork.create(base_branch="nonexistent")
+                branch = fork.create(base_branch="nonexistent")
```

Ruff (0.14.14)

[error] Local variable `branch` is assigned to but never used

Remove assignment to unused variable `branch`

(F841)